architectures/lib/commands/check-repositories.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+# © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
+
+- name: Check for old repositories
+  hosts: all
+
+  tasks:
+  - assert:
+      that: >
+        tpa_2q_repositories is not defined
+        or tpa_2q_repositories == []
+      msg: >-
```

Comment:
We should maybe use `fail_msg` to make it clear that it's a msg on failure of the assert. `msg` is an alias of `fail_msg`